vip_hci/hci_dataset.py

```diff
@@ -1183,13 +1183,13 @@ def rescale(self, scale, imlib='ndimage', interpolation='bicubic',
         self.cube = cube_px_resampling(self.cube, scale, imlib, interpolation,
                                        verbose)
 
-    def save(self, path, precision=np.float32):
-        """ Writing to FITS file. If self.angles is present, then the angles
-        are appended to the FITS file.
+    def export(self, path, precision=np.float32):
```

Comment:
Then let's call this method `write_fits` to be consisten with the current function names. Would it be possible to do the same, e.g. renaming `save` to `write_fits` and adding the new `save` method in the HCIFrame class?

Reply:
👍